Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chrome/ChromiumSecurityWarningsRegistryHelper.cs

```diff
@@ -22,7 +22,7 @@
 namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
 {
   /// <summary>
-  /// Responsible for interacting with the Registry (reading or manipulating the <CommandLineFlagSecurityWarningsEnabled/> entry for Chrome).
+  /// Responsible for interacting with the Registry (reading or manipulating the <see cref="c_securityWarningsPolicyName"/> entry for Chrome).

```

Comment:
Okay, that's a new problem. private fields in summaries are not helpful. We want to know the value, placed inside `<c>...</c>` tags